### PR TITLE
Feat/followers stats

### PR DESCRIPTION
**Pull request by Adesojisouljay**
### What does this PR?
In this PR, I have added stats for followers, users can now see how how active their following and followers are.
There is also a filter dropdown option where user can filter friends based on how active they are.
For example, you can filter friends that are active recently, this month, this year, one year ago or more than one year.

[Github issue](https://github.com/ecency/ecency-vision/issues/1386)

https://github.com/ecency/ecency-vision/assets/88087890/b1139fea-e448-4c4c-b0ad-1a3f76f80ebc

### CODE REVIEW

File: src/common/components/friends-filter/index.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            <div className="filter-options">
+            {(() => {
+            let dropDownConfig: any;
+            dropDownConfig = {
+                history: "",
+                label: label ? label : "Filter",
+                items: [
+                {
+                    label: <span>All</span>,
+                    onClick: () => {
```
Comment (feruzm) [inline src/common/components/friends-filter/index.tsx:22]: @Adesojisouljay labels should be translated

File: src/common/components/friends/index.tsx [hunk 2]
```diff
@@ -173,6 +190,141 @@ export class List extends BaseComponent<ListProps, ListState> {
...
+      const lastSeenTime = new Date(this.formatTimeDIfference(item.lastSeen));
+
+      const timeDifference = currentTime.getTime() - lastSeenTime.getTime();
+
+      const daysDifference = Math.ceil(timeDifference / (1000 * 3600 * 24));
+      const monthsDifference = Math.ceil(daysDifference / 30);
+      const yearsDifference = Math.ceil(daysDifference / 365);
+
+      return (
+        (type === "Recently" && daysDifference < 7) ||
```
[the remark this answers is not in the document]
Comment (feruzm) [reply src/common/components/friends/index.tsx:214]: @Adesojisouljay Also include description and quick video into PR description, please!

[the remark this answers is not in the document]
Comment (Adesojisouljay) [reply src/common/components/friends/index.tsx:214]: Noted, will do that,
Thanks

**Review by feruzm — APPROVED** [review]
lgtm

**Pull request merged**